src/sql/azdata.d.ts

```diff
@@ -4645,15 +4645,10 @@ declare module 'azdata' {
 		}
 
 		export interface INotebookMetadata {
-			kernelspec: IKernelInfo;
+			kernelspec?: IKernelSpec;
 			language_info?: ILanguageInfo;
 			tags?: string[];
-		}
-
-		export interface IKernelInfo {
```

Comment:
So unfortunately public api's like this we shouldn't just be removing interfaces. What we'll need to do here is 

- mark this as @deprecated 
- Modify INotebookMetadata to type kernelspec as `IKernelInfo | IKernelSpec`